src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -77,6 +82,9 @@ const GENE_QUERY = gql`
   }
 `;
 
+type Transcript = Pick<FullTranscript, 'stable_id'> &
+  Pick<FullTranscript, 'product_generating_contexts'>;
```

Comment:
Graphql query says:

```
        product_generating_contexts {
          product_type
        }
```

So `Pick<FullTranscript, 'product_generating_contexts'>` is a lie.